tour/data-model-code.md

```diff
@@ -35,31 +38,21 @@ title: Data Model Code
             System.out.println("Gotham Police Department Persons of Interest:");
             // A Scanner is an extension of java.lang.Iterable so behaves just like one.
             for (Map.Entry<Key, Value> entry : scan) {
-                System.out.println("Key:" + entry.getKey());
-                System.out.println("Value:" + entry.getValue());
+                System.out.printf("Key : %30s  Value : %s\n", entry.getKey(), entry.getValue());
             }
         }
 ```
 
 The code above will print (timestamp will differ):
 ```commandline
 Gotham Police Department Persons of Interest:
-Key:id0001 hero:alias [] 1510084285325 false
-Value:Batman
-Key:id0001 hero:name [] 1510084285325 false
-Value:Bruce Wayne
-Key:id0001 hero:wearsCape? [] 1510084285325 false
-Value:true
-Key:id0002 hero:alias [] 1510084285325 false
-Value:Robin
-Key:id0002 hero:name [] 1510084285325 false
-Value:Dick Grayson
-Key:id0002 hero:wearsCape? [] 1510084285325 false
-Value:true
-Key:id0003 villain:alias [] 1510084285325 false
-Value:Joker
-Key:id0003 villain:name [] 1510084285325 false
-Value:Unknown
-Key:id0003 villain:wearsCape? [] 1510084285325 false
-Value:false
+Key : id0001 hero:alias [] 1511306370025 false            Value : Batman
```

Comment:
The timestamp and delete boolean makes the output noisy.  However trying to omit would make the code noisy/longer and harder to understand.  I can't think of any way to improve this though.